.github/workflows/gpu.yml

```diff
@@ -45,10 +57,18 @@ jobs:
           -DUMF_FORMAT_CODE_STYLE=ON
           -DUMF_DEVELOPER_MODE=ON
           -DUMF_BUILD_LIBUMF_POOL_DISJOINT=ON
+          -DUMF_BUILD_LIBUMF_POOL_SCALABLE=ON
+          -DUMF_BUILD_LIBUMF_POOL_JEMALLOC=ON
           -DUMF_ENABLE_POOL_TRACKING=ON
-
+          -DUMF_BUILD_LEVEL_ZERO_PROVIDER=${{matrix.level_zero_provider}}
+          
       - name: Build UMF
-        run: cmake --build ${{github.workspace}}/build -j $(nproc)
+        run: |
+          if [ "${{ matrix.os-name }}" = "WINDOWS" ]; then
+            cmake --build ${{github.workspace}}/build --config ${{matrix.build_type}} -j $(nproc)
```

Comment:
you don't have to `if` this - this line should work on both OSes I guess

Reply:
Done